source/application/DbgHelp/DbgHelp.cpp

```diff
@@ -41,22 +41,29 @@ namespace DbgHelp
         {
             s_symbol_paths.append(L";").append(buffer.data());
         }
-        if (GetEnvironmentVariableW(L"_NT_ALTERNATIVE_SYMBOL_PATH", buffer.data(), buffer.size()) != 0)
+        if (GetEnvironmentVariableW(L"_NT_ALTERNATIVE_SYMBOL_PATH", buffer.data(), buffer.size())
+            != 0)
         {
             s_symbol_paths.append(L";").append(buffer.data());
         }
         if (GetEnvironmentVariableW(L"SYSTEMROOT", buffer.data(), buffer.size()) != 0)
         {
-            s_symbol_paths.append(L";").append(buffer.data())
-                .append(L";").append(buffer.data()).append(L"\\system32")
-                .append(L";").append(buffer.data()).append(L"\\SysWOW64");
+            s_symbol_paths.append(L";")
+                .append(buffer.data())

```

Comment:
Is there a setting to align this with the `.` on the previous line?

Reply:
Nope.